routing/routing_tests/index_graph_test.cpp

```diff
@@ -361,4 +362,58 @@ UNIT_TEST(SerializeSimpleGraph)
     TEST_EQUAL(graph.GetJointId({2, 1}), Joint::kInvalidId, ());
   }
 }
+
+//      Finish
+// 0.0003    *---------*
+//           |         |
+//           |         |
+//           |         |
+//           |         |
+//           |         |
+// 0.0002    *         *
+//           |         |
+//           |         |
+// 0.00015   *    F0   *
+//            \       /
+//              \   /
+// 0.0001         *---F0-*----*
+//                            ^
+//                            |
+//                            F1
+//                            |
+//                            |
+// 0                          * Start
+//   0         0.0001       0.0002
+// Note 1. F0 are F2 are one-way one-segment features. F1 is one-way four segment feature
```

Comment:
I mean:

// F0 is a two-way feature with a loop and F1 is an one-way one-segment feature.